setup.cfg

```diff
@@ -46,6 +46,7 @@ exclude =
     build
     temp
     flash_notebooks
+    .git
```

Comment:
flake8 tries to check in the .git dir because my branch name `bugfix/338-setup.py` ends with .py (^^;)
```
./.git/refs/heads/bugfix/338-setup.py:1: [E999] SyntaxError: invalid syntax
./.git/refs/remotes/origin/bugfix/338-setup.py:1: [E999] SyntaxError: invalid syntax
./.git/logs/refs/heads/bugfix/338-setup.py:1: [E999] SyntaxError: invalid syntax
./.git/logs/refs/heads/bugfix/338-setup.py:1: [E225] missing whitespace around operator
...
```